launcher/ui/pages/global/LauncherPage.cpp

```diff
@@ -430,6 +436,19 @@ void LauncherPage::loadSettings()
             }
             idx++;
         }
+
+        //TODO: Don't make it hardcoded via strings
+        auto currentCatStyle = s->get("CatStyle").toString();
+        ui->themeComboBoxCat->addItem("BackgroundCat");
+        ui->themeComboBoxCat->addItem("Jinx");
+        if(currentCatStyle == "BackgroundCat")
+        {
+            ui->themeComboBoxCat->setCurrentIndex(0);
+        }
+        else
+        {
+            ui->themeComboBoxCat->setCurrentIndex(1);
+        }
```

Comment:
I would recommend making it default to BackgroundCat if it reads a non-existent CatStyle from the config.